tools/flake8_plugin.py

```diff
@@ -26,12 +26,20 @@
 
 S008_msg = "S008 Use stdlib datetime.timezone.utc instead of pytz.utc / pytz.UTC"
 
+S009_msg = "S009 Do not import requirements-dev packages into production code."
+S009_modules = frozenset(
+    ("django_stubs_ext", "pytest", "responses", "time_machine", "selenium", "psutil")
+)
```

Comment:
there's quite a few more modules than this -- one can generate the list from the frozen file using `importlib.metadata`

Reply:
hmmm also `django_stubs_ext` imports are fine if they are `TYPE_CHECKING` guarded

Reply:
> hmmm also django_stubs_ext imports are fine if they are TYPE_CHECKING guarded

There are ways to make many of the test dependencies 'safe' with patterns like:

```python
try:
   import responses
catch ImportError:
    responses = None
```

While this and `TYPE_CHECKING` blocks are possibilities, I think we should make those exceptional cases. I'm hoping to prevent accidents that currently cannot be caught by CI, and only get caught in production right now making them more expensive to fix.

> there's quite a few more modules than this -- one can generate the list from the frozen file using importlib.metadata

I can give that a try.